### ISSUE TITLE
`eslint-comments/no-unused-disable` false positive when disabling line with `eslint-comments/no-unused-disable`

### ISSUE THREAD

**Issue by DetachHead** [import-js/eslint-plugin-import#2733]
```ts
// eslint-disable-next-line eslint-comments/no-unlimited-disable
// eslint-disable-next-line
```
errors:
```
'eslint-comments/no-unlimited-disable' rule is disabled but never reported.eslint (eslint-comments/no-unused-disable)
Unexpected unlimited 'eslint-disable-next-line' comment. Specify some rule names to disable. (eslint-comments/no-unlimited-disable)
```

**Comment by DetachHead** [comment]
oops wrong repo
